element_session/export/nwb.py

```diff
@@ -74,44 +61,41 @@ def session_to_nwb(
 
     session_identifier = {
         k: v.isoformat() if isinstance(v, datetime.datetime) else v
-        for k, v in session_key.items()
-    }
+        for k, v in session_key.items()}
 
     nwbfile_kwargs = dict(
-        session_id="_".join(session_identifier.values()), identifier=str(uuid4()),
-    )
+        session_id="_".join(session_identifier.values()), identifier=str(uuid4()))
 
     session_info = (session.Session & session_key).join(session.SessionNote, left=True).fetch1()
 
     nwbfile_kwargs.update(
         session_start_time=session_info["session_datetime"].astimezone(
-            datetime.timezone.utc
-        )
-    )
+            datetime.timezone.utc))
 
     nwbfile_kwargs.update(session_description=session_info.get("session_note", ""))
 
     experimenters = list((session.SessionExperimenter & session_key).fetch("user"))
 
     nwbfile_kwargs.update(experimenter=list(experimenters) or None)
 
-    if HAVE_ELEMENT_ANIMAL and element_animal.subject.schema.is_activated():
-        from element_animal.export.nwb import subject_to_nwb
+    try:
+        subject_to_nwb = getattr(session._linking_module, 'subject_to_nwb')
+    except AttributeError:
+        subject_id = '_'.join((getattr(session._linking_module, 'Subject') & session_key).fetch1('KEY').values())
+        nwbfile_kwargs.update(subject=pynwb.file.Subject(subject_id=subject_id))
+    else:
```

Comment:
Instead of `try`/`except` and `getattr`, I would prefer something like: 

```python
subject_to_nwb = getattr(session._linking_module, 'subject_to_nwb', False)

if subject_to_nwb:
    nwbfile_kwargs.update(subject=subject_to_nwb(session_key))
else:
    subject_id = '_'.join((getattr(session._linking_module, 'Subject') & session_key).fetch1('KEY').values())
    nwbfile_kwargs.update(subject=pynwb.file.Subject(subject_id=subject_id))
```

Reply:
Sure - that looks equivalent to me